.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 dist
 lerna-debug.log
 package-lock.json
+yarn.lock
```

Comment:
remove both lockfiles from the gitignore

Reply:
done